### PR TITLE
feature/css-grid: add challenge and improve place-items challenge

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
change order of rulesets such that styling is applied
improve styling of container
add image-board challenge for training grid-template-areas:
https://codesandbox.io/s/github/neuefische/web-exercises/tree/css-grid/sessions/css-grid/image-board?file=/css/styles.css

### CODE REVIEW

**Review by ThomasF85 — APPROVED** [review]
Looks good!

File: sessions/css-grid/image-board/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
+    Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+}
+
+/* 
+- Use grid-template areas to place the images as shown in the challenges file
+- Assign an area to each image
+*/
```
Comment (stefankorder) [inline sessions/css-grid/image-board/css/styles.css:16]: This part should be written inside a readme file. You could take a look at the readme files of the objects and arrays challenges https://github.com/neuefische/web-exercises/pull/65.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Great challenge for our students. We have agreed to create a readme file for each challenge, check out my comment. My solution can be found under https://codesandbox.io/s/css-grid-imageboard-challenge-solution-zok7jb?file=/css/styles.css.

**Review by stefankorder — APPROVED** [review]
👍🏻

**Pull request merged**